thread/src/components/LoginForm.js

```diff
@@ -19,58 +21,109 @@ class LoginForm extends Component {
         this.props.loginUser({ email, password });
     }
 
+    logo(){
+      return(
+        <Logo />
+      );
+    }
+
+
     renderSignInButton() {
         if (this.props.loading) {
             return (
-                <Button disabled full style={{ marginTop: 25 }}>
+                <Button rounded warning style={{ marginTop: 25 }}>
                     <Spinner size="small" color="#fff" />
                 </Button>
             );
         }
 
         return (
-            <Button full style={{ marginTop: 25 }} onPress={this.onSubmitPress.bind(this)}>
+            <Button rounded style={styles.buttonStyle} onPress={this.onSubmitPress.bind(this)}>
                 <Text>Login</Text>
             </Button>
         );
     }
 
     render() {
         return (
-            <Container style={{ backgroundColor: '#fff' }}>
-                <Content>
-                    <Form>
-                        <Item stackedLabel>
-                            <Label>Email</Label>
-                            <Input
-                                onChangeText={this.onEmailChange.bind(this)}
-                                value={this.props.email}
-                            />
-                        </Item>
-                        <Item stackedLabel last>
-                            <Label>Password</Label>
-                            <Input
-                                secureTextEntry
-                                onChangeText={this.onPasswordChange.bind(this)}
-                                value={this.props.password}
-                            />
-                        </Item>
-                    </Form>
+            <Container style={{ backgroundColor: '#8bc34a' }}>
+
+          {this.logo()}
+                <Content style={styles.viewStyle}>
+
+                        <Form>
+                              <Text style={styles.TextStyle}> Email </Text>
+                                <Item style={styles.inputStyle}>
+                                    <Content>
+                                      <Item>
+                                        <Input style={{color:'#424242'}}
```

Comment:
If we are going to heavily use styles, the best place for structuring is create the folder specifically for the component. Then we will have have two files inside it, one for styles and the other react stuff